src/classes/SObjectFieldWrapper.cls

```diff
@@ -0,0 +1,29 @@
+public class SObjectFieldWrapper {
+    private final List<SObjectField> fields;
+
+    public SObjectFieldWrapper(SObjectField field) {
```

Comment:
I might also give the class a new name at some point after you merge, just to try to get it to match the class names already in Nebula. Maybe 'QueryField' or something like that?